app/src/main/java/net/squanchy/SquanchyApplication.java

```diff
@@ -30,6 +30,7 @@ public void onCreate() {
         setupTracking();
         FirebaseDatabase.getInstance().setPersistenceEnabled(true);
         TypefaceManager.init();
+        applicationComponent().service();
```

Comment:
The thing to make sure of is just the nearITManager constructor being called.
How about having an init() method in the proximityService that does nothing? Just to make sure the constructor gets invoked